.github/workflows/build.yml

```diff
@@ -1,10 +1,34 @@
-name: Build Zed Nightly
+name: Build
 
 on:
-  schedule:
-    - cron: "0 0 * * *" # Runs every night at midnight UTC
+  workflow_call:
```

Comment:
This workflow is now a reusable one, called by the [stable.yml](https://github.com/deevus/zed-windows-builds/pull/36/files#diff-eddaa43b142a4a5fbb9297f1e7b3c59401c1f22790df62f04954baa4370e72e9) and [nightly.yml](https://github.com/deevus/zed-windows-builds/pull/36/files#diff-0d5658b415099a82c11c03a06ca4ec765b4003a1f4b2f3f1943980a882cf8aa6) workflows, which run on schedule instead.